src/types/components/AdminPortal/EventManagement/AgendaItemsDeleteModal/interface.ts

```
@@ -0,0 +1,16 @@
+/**
+ * Interface for the AgendaItemsDeleteModal component props.
+ *
+ * @property agendaItemDeleteModalIsOpen - Boolean to control modal visibility.
```

ESLint

[error] tsdoc-undefined-tag: The TSDoc tag "@property" is not defined in this configuration

(tsdoc/syntax)